### ISSUE TITLE
OSError: [WinError 10049] The requested address is not valid in its context

### ISSUE THREAD

**Issue by jagandeepbrar** [JagandeepBrar/Plex-Assistant-Bot#2]
```
OSError: [WinError 10049] The requested address is not valid in its context
client_socket.connect((constants.SOCKET_HOST, int(constants.SOCKET_PORT)))
File "C:\PLEXBOT\sonarr.py", line 43, in <module>
```

Problem opening the socket in the Radarr and Sonarr scripts on Windows machines. Related to using socket host 0.0.0.0 most likely. 

https://www.reddit.com/r/PleX/comments/an1lo8/plex_assistant_telegram_bot/eg46swu

**Comment by jagandeepbrar** [comment]
Fixed with pull request #3

**Comment by ghost** [comment]
how to solve WinError10049?
